src/main/java/org/gatorgradle/util/Read.java

```diff
@@ -0,0 +1,27 @@
+package org.gatorgradle.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Read {
```

Comment:
This is not a good solution -- remember the one-liner? That should be used instead, not in this method, but where this method is used. This method is also too specific -- it both reads the file, and then JSON-escapes it, which would definitely not work in many other situations you might want to use this method in; it's not general enough for a utility method. Also, this class does not have a good name. I would have suggested something like `FileUtil` to compliment `StringUtil`.

Reply:
Are you suggesting to directly use this one-line code at the JSON report building part? In that case, we don't even need to keep this class anymore, right?

Reply:
Exactly. Since the code is not complicated, the abstraction this class could provide is pretty useless. If you had many places that you would read a file, and perhaps wanted to ensure one place to change it (if they were all reading the same type of file, or something), then this solution would be good. But since that isn't the case, it's probably best to do as you said.